lib/abci/errors/wrapInErrorHandlerFactory.js

```diff
@@ -69,31 +69,20 @@ function wrapInErrorHandlerFactory(logger, isProductionEnvironment) {
           );
 
           if (!isProductionEnvironment) {
-            error = new VerboseInternalAbciError(error);
+            error = new VerboseInternalGrpcError(error);
           }
         }
 
-        const events = [];
-
-        const attributes = Object.entries(error.getTags())
-          .map(([key, value]) => new EventAttribute({ key, value, index: true }));
-
-        if (attributes.length > 0) {
-          events.push(new Event({
-            type: 'error',
-            attributes,
-          }));
-        }
+        const serializedError = cbor.encode({
+          error: {
```

Comment:
I think we don't need this key anymore, so just move message and data on upper level